soc/renesas/ra/ra4m1/Kconfig.soc

```diff
@@ -18,3 +18,27 @@ config SOC_SERIES
 
 config SOC
 	default "r7fa4m1ab3cfm" if SOC_R7FA4M1AB3CFM
+
+config OPTION_SETTING_MEMORY
+	bool "Option Setting Memory"
+	default y
+
+config OPTION_SETTING_LVDAS_DISABLE
+	bool "Disable voltage monitor 0 after reset"
+	default y
+	depends on OPTION_SETTING_MEMORY
+
+config OPTION_SETTING_HOCO_DISABLE
```

Comment:
Please refer to the devicetree definitions.

Reply:
https://github.com/zephyrproject-rtos/zephyr/blob/f643f3a6cf6bafd6338c090f4413e34eb3ed806c/dts/arm/renesas/ra/ra-cm4-common.dtsi#L39-L44

Please use this definition for determining HOCO frequency.